chain/network/src/routing/routing_table_actor.rs

```diff
@@ -89,6 +101,22 @@ impl RoutingTableActor {
             .get_ser::<u64>(ColLastComponentNonce, &[])
             .unwrap_or(None)
             .map_or(0, |nonce| nonce + 1);
+        /*
+         5: actix::sync::SyncArbiter<A>::start
+           at /home/pmnoxx/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-0.11.0-beta.2/src/sync.rs:122:23
+         6: near_network::routing::routing_table_actor::RoutingTableActor::new
+                   at ./src/routing/routing_table_actor.rs:104:34
+         7: cache_edges::RoutingTableTest::new
+                   at ./tests/cache_edges.rs:44:28
+         8: cache_edges::empty
+                   at ./tests/cache_edges.rs:183:20
+         9: cache_edges::empty::{{closure}}
+                   at ./tests/cache_edges.rs:182:1
+        10: core::ops::function::FnOnce::call_once
+
+
+               */
+        let edge_verifier_pool = SyncArbiter::start(4, || EdgeVerifierActor {});
```

Comment:
What is this?

Reply:
@bowenwang1996 That has been fixed. Some tests were not working due to now starting system with `System::new()`